tests/integration/test_elasticsearch.py

```diff
@@ -73,8 +74,12 @@ def test_domain_creation():
     assert_true(status['DomainStatus']['EBSOptions']['EBSEnabled'])
 
     # make sure we can fake adding tags to a domain
-    response = es_client.add_tags(ARN='string', TagList=[{'Key': 'SOME_TAG', 'Value': 'SOME_VALUE'}])
-    assert_equal(200, response['ResponseMetadata']['HTTPStatusCode'])
+    try:
+        response = es_client.add_tags(ARN='string', TagList=[{'Key': 'SOME_TAG', 'Value': 'SOME_VALUE'}])
+        assert_equal(200, response['ResponseMetadata']['HTTPStatusCode'])
+    except ClientError as e:
```

Comment:
After the rebase on master I am getting strange random failures in tests for other services:
- elasticsearch
- test_web_ui.test_infra_graph_generation

Reply:
@mberlanda builds are passing against latest `master`. Can we remove the `try-except` and `print` statements here - I can then update the base image with your change (updating to `0.14.5`). Thanks